routers/user.controller.js

```diff
@@ -85,14 +89,44 @@ class UserController {
       }).validate(req.body);
 
       if (validationError) {
-        throw new ApiError(400, 'Bad requiest', validationError);
+        throw new ApiError(400, 'Bad request', validationError);
       }
 
       next();
     } catch (e) {
       errorHandler(req, res, e);
     }
   }
+
+  //========================================
+
+  async userLogout(req, res) {
+    try {
+      const { activeToken, user } = req;
+
+      user.tokens = user.tokens.filter(
+        (tokenRecord) => tokenRecord.token !== activeToken,
+      );
+
+      await user.save();
+
+      res.status(204).send();
+    } catch (err) {
+      errorHandler(req, res, err);
+    }
+  }
+
+  //==========================
+
+  async userCurrent(req, res) {
+    try {
+      const { _id, name, email, familyId } = req.user;
+
+      res.status(200).send({ _id, name, email, familyId });
```

Comment:
We're sending `id` in sign-in/sign-up as user id. Please keep up with that convention. Also could you extract user response object composition into a function? It would solve such problems and then sign-in/sign-up/current would use that function